doc/REL_NOTES

```diff
@@ -51,6 +51,10 @@ This Release
          - Add VMware for network ordering (BZ1024)
          - Fixes for byte order selecting.
          - Add ASCII and MacRoman as aliases for the ISO-8859-1 codeset
+         - Fix for bidirectional IIOP where the client creates a callback IOR with alternate
```

Comment:
Hi Nick and Phil, I tested this (Phils fork) as well with multi-homed omniorb and IIOP.NET clients. Have been only quick tests so far but looking to integrate this branch in multiple development machines to have more use cases.